### ISSUE TITLE
Messages Table

### ISSUE THREAD

**Issue by khashish** [eventuate-tram/eventuate-tram-core#6]
Is there a reason for limiting the payload of a message to 1000 characters only?

https://github.com/eventuate-tram/eventuate-tram-core/blob/master/postgres/initialize-database.sql

**Comment by cer** [comment]
No. The choice is arbitrary. We could certain increase it.

What message size are you using?

**Comment by khashish** [comment]
It could maybe saved on the database side with type “text” instead of ”varchar” or is there a reason that its varchar?

**Comment by feanor777** [comment]
Yep, the same from my side, our JSON is much longer than 1000 chars.
"text" would be much better I think.

**Comment by feanor777** [comment]
And as I understood in saga_instance table, saga_data_json column is varchar(1000) for the same, reason?
